### PR TITLE
Update controller-runtime to v0.15.0 & client-go to v0.28.0-alpha.2

### PR DESCRIPTION
**Pull request by austincunningham**
…double the requests and limits on the operator deployment

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ commands:
 jobs:
   install-operator:
     docker:
-      - image: cimg/go:1.18
```
Comment (eguzki) [inline .circleci/config.yml:37]: Is Golang 1.20 ready to be used in the productization pipeline?

Comment (austincunningham) [reply .circleci/config.yml:37]: hmm good question , I'm not sure , if not we will have to put this on hold for a while.

**Comment by MStokluska** [comment]
@austincunningham is this still needed?

**Comment by austincunningham** [comment]
@MStokluska no @carlkyrillos will have updated this

**Pull request closed without merging**